### PR TITLE
chore: fix flutter 3 issues

### PR DESCRIPTION
**Pull request by maeddin**

### CODE REVIEW

File: lib/src/glass_container.dart [hunk 1]
```diff
@@ -403,7 +403,7 @@ class GlassContainer extends StatelessWidget {
       current = ClipOval(child: current);
     } else {
       assert(borderRadius != null);
-      current = ClipRRect(borderRadius: borderRadius, child: current);
+      current = ClipRRect(borderRadius: borderRadius!, child: current);
```
Comment (bharat-1809) [inline lib/src/glass_container.dart:406]: let's use conditional/null-aware(`??`) operators instead of null-assertion(`!`)

File: example/pubspec.lock
Comment (bharat-1809) [inline example/pubspec.lock:1]: any reason why the change in `pubspec.lock` was required? can it be done by changing the versions in `pubspec.yaml`

File: lib/src/glass_container.dart (hunk 1 above)
Comment (maeddin) [reply lib/src/glass_container.dart:406]: I did not want to remove the assertion and therefore a conditional operator was not necessary. As far as the `assert` should be removed, you can of course use the `?? BorderRadius.zero`.

File: example/pubspec.lock
Comment (maeddin) [reply example/pubspec.lock:1]: That was my mistake for committing this. I have not changed any versions. The `pubspec.lock` was apparently just not synchronized with the `pubspec.yaml`.

**Comment by bharat-1809** [comment]
@all-contributors please add @maeddin for maintenance and code
